Cilsil/Utils/ProgramState.cs

```diff
@@ -176,6 +201,20 @@ public void SaveNodeOffset(CfgNode node, ProgramStack previousStack)
                             .Node, false);
                 }
             }
+
+            // If instruction count between leave is 0, we make it a special case and don't count it as off leave.
```

Comment:
off leave is confusing -- as before, please define what you mean and come up with a more descriptive term

Reply:
Done